fv_be/app/views/base_views.py

```diff
@@ -0,0 +1,24 @@
+from rest_framework.response import Response
+from rules.contrib.rest_framework import AutoPermissionViewSetMixin
+
+
+class BaseViewSetMixin(AutoPermissionViewSetMixin):
```

Comment:
This looks great! My only suggestion is to give this mixin a more descriptive name, like FVPermissionViewSetMixin or something

Reply:
I agree, the rest LGTM!

Reply:
Updated naming in https://github.com/First-Peoples-Cultural-Council/fv-be/pull/20/commits/4a6d78bdbb381cfb7cff5dcd4a7cd8df7227df02, thanks!